sql/moz-fx-data-shared-prod/telemetry_derived/urlbar_clients_daily_v1/query.sql

```diff
@@ -39,7 +39,7 @@ WITH combined_urlbar_picked AS (
     experiments,
     app_version,
     normalized_channel,
-    country,
+    IFNULL(country, '??') AS country,
```

Comment:
This looked like the only other change needed to pull from `clients_daily_joined_v1` instead of the view